tests/integration/lxd/test_lxd_provider.py

```diff
@@ -40,16 +42,26 @@ def test_create_environment(installed_lxd, instance_name):
     assert test_instance.exists() is False
 
 
-def test_launched_environment(installed_lxd, instance_name, tmp_path):
+@pytest.mark.parametrize(
+    "alias",
+    [
+        BuilddBaseAlias.BIONIC,
+        BuilddBaseAlias.FOCAL,
+        BuilddBaseAlias.JAMMY,
+        BuilddBaseAlias.KINETIC,
+        BuilddBaseAlias.LUNAR,
+    ],
```

Comment:
Are we intentionally not testing on Xenial?

If we intend to test on all supported base aliases it may be better to do:

```suggestion
    "alias", list(BuilddBaseAlias),
```

so the tests automatically catch any new bases added.

Reply:
`xenial` was conspicuously missing and untested in craft-providers (except one integration test).  It would raise an exception when trying to launch it using the Provider classes.

It's still available for LXD, but not for Multipass.  `lpcraft` still uses `xenial`.  

I'll add support for it and use your suggestion too!

Reply:
After adding `xenial`, I'm seeing failures related to the version of `snapd` being too old.

Reply:
Welp, now we know! Thank you!